src/pages/Homepage/iwd/IWDHomepage.vue

```diff
@@ -11,7 +11,6 @@
 		>
 			<kv-responsive-image
 				:images="heroImageSet"
-				alt=""
```

Comment:
Not needed, kv responsive wont even pass attributes down.

Reply:
@mcstover @eddieferrer A small thing, but we should always have some alt tag on images. `alt=""` tells a screenreader that this is a decorative image and to ignore it. Having no alt attribute [makes some screenreaders announce the image URL](https://www.w3.org/WAI/tutorials/images/decorative/). 

kv-responsive-image v-binds $attrs to the `img` inside. We could add a non-required alt prop, and if there's no prop set alt to "" on the img inside. Though I kindof like reinforcing the always alt pattern so it's remembered for regular images.

Reply:
This is a good callout. I think we just got caught up in the rush.

Reply:
@ryan good point, I was incorrect about my comment above. Ill revert this